tests/test_install_mender_sh.py

```diff
@@ -230,7 +230,7 @@ def test_configure(
             f"curl http://{SCRIPT_SERVER_ADDR}:{SCRIPT_SERVER_PORT}/install-mender.sh | bash -s -- mender-configure"
         )
 
-        check_installed(generic_debian_container, "mender-client")
+        check_installed(generic_debian_container, "mender-auth")
```

Comment:
`mender-configure` depends on `mender-update`, so it will be good to check for this one too :+1: